extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -36,6 +35,7 @@ import {
   useState,
 } from '../../../src/preact';
 import {useMountEffect, useResourcesNotify} from '../../../src/preact/utils';
+import {useStyles} from './autoplay.jss';
```

Comment:
@samouri I'm not entirely sure if it's a good idea to keep this per-feature. I'm not sure if there's a restriction on one `useStyles` per component?

Reply:
discussed down below -- multiple jss files are fine!